sync-pingdom-ec2-security-groups.py

```diff
@@ -92,20 +93,24 @@ def get_target_permissions(self):
     def get_ips(self):
         r = requests.request('GET', self.whitelist, timeout=10)
         r.raise_for_status()
-        return set(r.iter_lines())
+        return set(r.iter_lines(decode_unicode=True))
 
     def create_permission(self, ip):
         return Permission(self.protocol, '{0}/32'.format(ip), self.from_port, self.to_port)
 
     def configure_permissions(self, permissions):
-        ec2 = boto3.resource('ec2')
+        ec2 = boto3.resource('ec2', region_name=self.region)
         for sg_id in self.security_groups:
             sg = PingdomSecurityGroup(ec2.SecurityGroup(sg_id))
             sg.update_permissions(permissions)
 
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument(
+        '--region',
+        default='us-east-1',
+        help='The AWS region where the security groups are located')
```

Comment:
Hmm, our SGs are in `eu-west-1` and prior to this changed it "just worked", presumably because this region is configured in my `~/.aws/config`. I'd be nice if we can retain that functionality. It seems setting the default to `None` does the trick.